docs/source/getting-started.rst

```diff
@@ -59,3 +59,29 @@ Below is a simple example:
 
 Full example code is available `here <https://github.com/EthWorks/useDapp/tree/master/packages/example>`_.
 
+Creating new hook
+------------
+
+Creating a custom hook with use of our core hooks is very easy, as example let's write a *useEtherBalance* hook:
+
+.. code-block:: javascript
+
+  export function useEtherBalance(address: string | Falsy) {
+    const multicallAddress = useMulticallAddress() // gets address of multicall contract
+    const getEthBalanceCall = address &&
+      MultiCallABI.encodeFunctionData('getEthBalance', [address])
+    //encodes data for a function call
+
+    const etherBalance = useChainCall(
+      getEthBalanceCall && multicallAddress &&
+        { address: multicallAddress, data: getEthBalanceCall }
+    ) //makes a chain call
+
+    return { etherBalance: etherBalance !== undefined
+      ? BigNumber.from(etherBalance)
+      : undefined } //returns balance
```

Comment:
You will need useMemo an explanation why you need useMemo.

Reply:
Should I wrap the return in useMemo? should it be done in the hook or just in docs?

Reply:
We moved to useTokenBalance and it's not relevant anymore